actors/market/src/types.rs

```diff
@@ -71,11 +71,9 @@ pub struct PublishStorageDealsReturn {
     pub valid_deals: BitField,
 }
 
-// Changed since V2:
-// - Array of Sectors rather than just one
-// - Removed SectorStart
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
 pub struct VerifyDealsForActivationParams {
+    // Deals to verify, grouped by sector.
```

Comment:
And elsewhere:
"///" is a doc comment allowing comments on structs or their fields to be displayed on hover in IDEs